### PR TITLE
[906] Create dv_review environment

### PR DESCRIPTION
**Pull request by saliceti**
## Context
In order to test infrastructure changes, it's useful to deploy the app to a dev cluster

## Changes proposed in this pull request
Create the dv_review environment

## Guidance to review
- Deploy a dev cluster
- Deploy app using: make dv_review terraform-apply CLUSTER=clusterx

## Link to Trello card
https://trello.com/c/WTWtPR8H

### CODE REVIEW

**Comment by saliceti** [comment]
[Spike: application log shipping](https://trello.com/c/WTWtPR8H/906-spike-application-log-shipping)

File: Makefile [hunk 1]
```diff
@@ -16,6 +16,14 @@ review: test-cluster
 	$(eval export TF_VAR_environment=${ENVIRONMENT})
 	$(eval include global_config/review.sh)
 
+dv_review: dev-cluster
```
Comment (ollietreend) [inline Makefile:19]: What does `dv_review` mean in this context? I'm not sure what `dv` means.

**Review by RMcVelia — APPROVED** [review]
deployed (and destroyed) to dev cluster5 ok

File: Makefile (hunk 1 above)
Comment (RMcVelia) [reply Makefile:19]: dev. It's a review app deployed to a dev cluster. Standard review apps go to the test cluster.

Comment (saliceti) [reply Makefile:19]: Yes the clusters are dv, ts and pd. So dv is the dev cluster, not to confuse with ittms "dev" app environment.

**Pull request merged**